src/components/Articles.jsx

```diff
@@ -1,21 +1,43 @@
 import React, { useEffect,useState,useContext } from "react"
+import {useParams} from "react-router-dom"
 import axios from 'axios'
+import { getArticles } from "../utils/api"
 
 const Articles = () => {
+   
     const[articles, setArticles] = useState([])
```

Comment:
Could you please add and use a loading state, so that you can show a loading message/icon while the articles are loading, that way if the user has slow internet- they still know articles are on the way! 🐌